src/kaocha/type/clojure/spec/alpha/ns.clj

```diff
@@ -0,0 +1,29 @@
+(ns kaocha.type.clojure.spec.alpha.ns
+  (:require [clojure.spec.alpha :as s]
+            [clojure.spec.test.alpha :as stest]
+            [kaocha.core-ext :as core]
+            [kaocha.hierarchy :as hierarchy]
+            [kaocha.testable :as testable]
+            [kaocha.type :as type]
+            [kaocha.type.clojure.spec.alpha.fdef :as type.fdef]
+            [kaocha.type.ns :as type.ns]))
+
+(def ->testable (partial type.ns/testable :kaocha.type/ns))
+
+(defmethod testable/-load :kaocha.type/ns-spec-fdefs [testable]
```

Comment:
`ns-spec-fdefs` doesn't seem to be referenced anywhere?

Reply:
Probably just a WIP issue. While we're on the topic, do you think this is an appropriate name for "all the clojure.spec fdefs in the namespace?"